drivers/sensor/weplatform/weplatform.c

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
I think it is clearer if you split the i2c & spi like so: https://github.com/zephyrproject-rtos/zephyr/tree/main/drivers/sensor/iis2dh

Reply:
Done